server/services/prisonerListService.ts

```diff
@@ -0,0 +1,12 @@
+import { HmppsAuthClient, PrisonerSearchClient } from '../data'
+import EducationAndWorkPlanClient from '../data/educationAndWorkPlanClient'
+import CiagInductionClient from '../data/ciagInductionClient'
+
+export default class PrisonerListService {
+  constructor(
+    private readonly hmppsAuthClient: HmppsAuthClient,
+    private readonly prisonerSearchClient: PrisonerSearchClient,
+    private readonly educationAndWorkPlanClient: EducationAndWorkPlanClient,
+    private readonly ciagInductionClient: CiagInductionClient,
+  ) {}
```

Comment:
This is the new service class. I'll start to flesh out a service method in my next PR